pom.xml

```diff
@@ -65,6 +55,11 @@
         </dependency>
 
         <!-- test dependencies -->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-text</artifactId>
+            <version>1.7</version>
```

Comment:
Should this be scoped to test? (<scope>test</scope>)

Reply:
good catch, will fix